src/components/Counter.js

```diff
@@ -1,59 +1,62 @@
-import { connect } from 'react-redux';
-import { increment, decrement } from '../actions';
+import { connect } from "react-redux";
+import { increment, decrement } from "../actions";
 
 class Counter extends Component {
-    incrementIfOdd = () => {
-        // Stretch Problem: Implement an increment function that
-        // only increments if the counter value is odd
-    };
+  incrementIfOdd = () => {
+    // Stretch Problem: Implement an increment function that
+    // only increments if the counter value is odd
+  };
 
-    incrementAsync = () => {
-        // Stretch Problem: Implement an increment function that
-        // increments after waiting for one second
-    };
+  incrementAsync = () => {
+    // Stretch Problem: Implement an increment function that
+    // increments after waiting for one second
+  };
 
-    render() {
-        // Fill in the two button onClick methods
-        // Upon clicking these buttons, the count
-        // should decrement or increment accordingly
-        return (
-            <p>
-                Clicked: {this.props.count} times
-                <button onClick={() => /* Fill me in */ }>
-                    +
-                </button>
-                <button onClick={() => /* Fill me in */ }>
-                    -
-                </button>
-                 {/* Uncomment these button tags if you got
+  render() {
+    // Fill in the two button onClick methods
+    // Upon clicking these buttons, the count
+    // should decrement or increment accordingly
+    return (
+      <p>
+        Clicked: {this.props.count} times
+        <button onClick={() => this.props.increment(1) /* Fill me in */}>
+          +
+        </button>
+        <button onClick={() => /* Fill me in */ this.props.decrement(1)}>
+          -
+        </button>
+        {/* Uncomment these button tags if you got
                 around to implementing the extra credit functions */}
-                {/* <button onClick={this.incrementIfOdd}>
+        {/* <button onClick={this.incrementIfOdd}>
                     Increment if odd
                 </button>
                 <button onClick={this.incrementAsync}>
                     Increment async
                 </button>  */}
-            </p>
-        );
-    }
+      </p>
+    );
+  }
 }
 
-// The mapStateToProps function specifies which portion of the 
-// state tree this component needs to receive. In this case, 
+// The mapStateToProps function specifies which portion of the
+// state tree this component needs to receive. In this case,
 // since our redux store is only storing the value of the count,
 // this component receives the whole state. In a more complex
 // redux application, though, it would receive only the relevant
 // parts it needs from the state object.
-const mapStateToProps = (state) => {
-    return {
-        count: state.count
-    };
+const mapStateToProps = state => {
+  return {
+    count: state.count
+  };
```

Comment:
You could shorten this with object destructuring. For example:
```javascript
const mapStateToProps = ({ count }) => { 
  return {
    count: count
  };
}
```
or shorter:
```javascript
const mapStateToProps = ({ count }) => { 
  return {
    count
  };
}
```
or even shorter, using an implicit return:
```javascript
const mapStateToProps = ({ count }) => ({ count });
```